src/main/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelService.kt

```diff
@@ -59,22 +60,9 @@ class EndretUtbetalingAndelService(
             hentEndredeUtbetalingAndeler(behandling.id)
                 .filter { it.id != endretUtbetalingAndelRequestDto.id }
 
-        val gyldigTomEtterDagensDato =
-            beregnGyldigTomIFremtiden(
-                andreEndredeAndelerPåBehandling = andreEndredeAndelerPåBehandling,
-                endretUtbetalingAndel = endretUtbetalingAndel,
-                andelTilkjentYtelser = andelTilkjentYtelser,
-            )
+        val tomDatoPåEndretUtbetalingAndel = endretUtbetalingAndel.tom ?: throw Feil("Tom dato må være satt på dette tidspunktet.")
```

Comment:
Kan denne feilmeldingen nå bruker? Isåfall burde vi kanskje ha noe mer forklarende 🤔

Reply:
Valgte aktivt å ikke kaste funksjonell feil her, siden det er noe galt med koden hvis vi kan komme bort hit, men tom fortsatt er null 😭